### PR TITLE
project details update

### PR DESCRIPTION
**Pull request by davidclaphan**
first pass updates to projects

### CODE REVIEW

File: src/constants/projects.ts [hunk 1]
```diff
@@ -1,85 +1,24 @@
...
-        projectImage: projectImage,
-        projectName: "ML Music Classification",
-        projectDescription: "This is a long description to see what it would look like if we had a longer description for one of our projects, like our AI/ML Capstone Project. I want to get this to about 250 chars and 50 words to see. Look at this, that is just about perfect!!!!",
-        projectSkills: ["AI/ML", "React.js", "Flask"],
+        projectImage: mlCapstone,
+        projectName: "ML Music Genre Prediction App",
+        projectDescription: "A machine learning project, our system utilizes convolutional neural networks and signal processing \
+        to classify audio samples into genres. The user can then interact with the model by submitting a YouTube URL, and the program will \
+        output the predicted genre matches for the provided audio.",
+        projectSkills: ["AI/ML", "React.js", "Python", "TensorFlow"],
```
Comment (arse0039) [inline src/constants/projects.ts]: We can add Flask to this list as well.

File: src/constants/projects.ts [hunk 2]
```diff
@@ -1,85 +1,24 @@
...
-        projectDescription: "Check out our project",
-        projectSkills: ["React.js", "Express", "node.js"],
-        github: 'https://pythontutor.com',
-        url: 'https://pythontutor.com'
+        projectImage: uffdaGif,
+        projectName: "Database Management Web Application",
+        projectDescription: "A database management tool meant to be used by database administrators to manage a \
+        community-driven recreation service. A SQL database holds information related to 5 different entities and supports \
+        basic CRUD operations for all entities.",
+        projectSkills: ["React.js", "SQL", "node.js"],
```
Comment (arse0039) [inline src/constants/projects.ts]: And JavaScript to this one.

**Review by arse0039 — DISMISSED** [review]
Looks good! Just a couple skills additions we can add, otherwise, good to go!

File: src/constants/projects.ts (hunk 1 above)
Comment (davidclaphan) [reply src/constants/projects.ts]: done

File: src/constants/projects.ts (hunk 2 above)
Comment (davidclaphan) [reply src/constants/projects.ts]: done

**Review by arse0039 — APPROVED** [review]
Love these images!

**Pull request merged**